latentnetworks/vimure
# [R] Add unittests for synthetic functions

Closes #18 

## How to validate this MR?

- [ ] [Install the package in the development mode](https://github.com/latentnetworks/vimure/blob/18-vimure-v01-r-write-test_syntheticr/src/R/README.md#setup-development-mode)
- [ ] Run the unittests
```r
install.packages("devtools")
devtools::test()
```

You should see output similar to the image below:

![image](https://user-images.githubusercontent.com/22843858/166327117-fd6f85a7-388e-4fc9-8928-1d86aaaa4de9.png)


Note: It is not possible to test `BaseNetwork` and `BaseSyntheticNetwork` because they are abstract classes.
